src/cli/commands/pluginGraph.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import {Command, flags} from "@oclif/command";
+import stringify from "json-stable-stringify";
+
+import type {Graph} from "../../core/graph";
+import createGitGraph from "../../plugins/git/cloneGitGraph";
+import createGithubGraph from "../../plugins/github/fetchGithubGraph";
+
+export default class PluginGraphCommand extends Command {
+  static description = "create the contribution graph for a single plugin";
+
+  static args = [
+    {
+      name: "repo_owner",
+      required: true,
+      description: "owner of the GitHub repository for which to fetch data",
+    },
+    {
+      name: "repo_name",
+      required: true,
+      description: "name of the GitHub repository for which to fetch data",
+    },
+  ];
+
+  static flags = {
+    plugin: flags.string({
+      description: "plugin whose graph to generate",
+      required: true,
+      options: ["git", "github"],
```

Comment:
also factor to a constant somewhere

Reply:
also, I'm sure you loved seeing that "multiple" [is an option that defaults to false](https://oclif.io/docs/flags.html)

Reply:
I did enjoy seeing that. I considered using that instead of `strict: false` for `combine`, but I believe that that would require writing `-f graph1 -f graph2 -f graph3 …`, instead of the more acceptable `-f graph1 graph2 graph3`, which is in turn inferior to just `graph1 graph2 graph3`.

Reply:
:+1: